src/main/java/seedu/address/model/StateManager.java

```diff
@@ -29,6 +32,11 @@ public static StateManager getInstance() {
         return instance;
     }
 
+    // Updates the Model in StateManager
```

Comment:
Not sure if this commenting matches the coding standard, should this use Javadoc commenting?